validation/src/main/scala/hmda/validation/engine/ts/quality/TsQualityEngine.scala

```diff
@@ -0,0 +1,17 @@
+package hmda.validation.engine.ts.quality
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.api.ValidationApi
+import hmda.validation.engine.ts.TsCommonEngine
+import hmda.validation.rules.ts.quality._
+
+trait TsQualityEngine extends TsCommonEngine with ValidationApi {
```

Comment:
You might need to throw this into the `TsEngine` trait.  I would double-check with @jmarin though.